azkaban-web-server/src/main/java/azkaban/webapp/servlet/UserUtils.java

```diff
@@ -0,0 +1,27 @@
+package azkaban.webapp.servlet;
+
+import azkaban.user.Permission;
+import azkaban.user.Role;
+import azkaban.user.User;
+import azkaban.user.UserManager;
+import azkaban.webapp.AzkabanWebServer;
+
+
+public final class UserUtils {
+  private UserUtils() {
+
+  }
+
+  public static boolean
```

Comment:
It's not currently package private.
However, I will move it to azkaban.user in module azkaban-common.